Bank Management/src/CreateAccount.java

```diff
@@ -120,10 +120,10 @@ public void actionPerformed(java.awt.event.ActionEvent evt) {
         );
         layout.setVerticalGroup(
             layout.createParallelGroup(javax.swing.GroupLayout.Alignment.LEADING)
-            .addGroup(layout.createSequentialGroup()
+            addGroup(layout.createSequentialGroup()
                 .addContainerGap()
```

Comment:
There is a missing parenthesis in the addGroup method call. It should be addGroup(layout.createSequentialGroup().